pkg/cmd/step/verify/test_data/preinstall/no_kaniko_or_terraform/jx-requirements.yml

```diff
@@ -14,7 +14,7 @@ ingress:
   namespaceSubDomain: -jx.
   tls:
     email: ""
-    enabled: false
+    enabled: true
```

Comment:
I suppose that there is already a test to cover the case when TLS is enabled.

Reply:
I'm not sure I follow - this test is for TLS enabled and I added one for TLS disabled.